### PR TITLE
fix: extended NpmProxyITCase to test individual repository port

### PR DESCRIPTION
**Pull request by andpopov**
closes: #1055
Hi @olenagerasimova.
Could you please to take a look at this PR?

### CODE REVIEW

File: src/test/java/com/artipie/npm/NpmProxyITCase.java [hunk 1]
```diff
@@ -90,14 +94,14 @@ void installFromProxy() throws Exception {
...
         );
         this.containers.assertArtipieContent(
             "artipie-proxy",
             "Package was not cached in proxy",
-            String.format(
-                "/var/artipie/data/my-npm-proxy/%s/-/%s-1.0.1.tgz",
-                NpmProxyITCase.PROJ, NpmProxyITCase.PROJ
+            String.format("/var/artipie/data/%s", repo)
+            .concat(
+                String.format("/%s/-/%s-1.0.1.tgz", NpmProxyITCase.PROJ, NpmProxyITCase.PROJ)
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/npm/NpmProxyITCase.java:104]: I'd suggest simplifying this expression by using only `String.format` without any concatenation

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@andpopov thansk, one suggestion

File: src/test/java/com/artipie/npm/NpmProxyITCase.java (hunk 1 above)
Comment (andpopov) [reply src/test/java/com/artipie/npm/NpmProxyITCase.java]: @olenagerasimova thanks for suggestion!
I have added according fix for it. 
Could you please to review it again?

**Review by olenagerasimova — APPROVED** [review]
@andpopov thanks, merging

**Pull request merged**